lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -57,6 +59,11 @@
  */
 class ClassMetadataFactory extends AbstractClassMetadataFactory
 {
+    private const SEQUENCE_NAME_NEEDS_TRUNCATION = [
```

Comment:
Why not use a list here, and `in_array` below?

Reply:
Me micro-optimizing stuff. A lookup by key is faster than searching the array for a value. Then again, the array only has two elements. 🙈 

What would be your preference?

Reply:
Class metadata is cached , so you don't have to over-optimize here, that's for sure!

Reply:
`in_array()` it is